src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/DatabaseTreeNode.cs

```diff
@@ -19,21 +19,30 @@ public DatabaseTreeNode(ServerNode serverNode, string databaseName): this()
         {
             Parent = serverNode;
             NodeValue = databaseName;
-            Database db = new Database(serverNode.GetContextAs<SmoQueryContext>().Server, this.NodeValue);
+            var db = new Database(serverNode.GetContextAs<SmoQueryContext>().Server, this.NodeValue);
```

Comment:
>var [](start = 12, length = 3)

+1 on this var usage. The convention has been to avoid the use of var, though personally for local method specific and specially loop specific variable there are ok from my point of view. Especially when followed by a new or a cast expression that makes type abundantly clear.

There are a lot of articles on internet discussing the pros and cons :) so I will not go into those here, but we will need to be consistent within our team.

Reply:
I try to use var whenever it's in a type-obvious situation but i also don't really mind inconsistencies across a code base with it. As long as the code is expressive, most of the time it's ok.

Reply:
Yeah our rule had been "unless you can see the "new Xyz" statement don't use var, but here it's clear (RHS is a new Database statement).